server/UserRegistration.java

```diff
@@ -0,0 +1,357 @@
+
+        System.out.println("\n Your new password is:" + " " + newPassword);
+        return newPassword;
+    }
+
+
+    public static boolean passwordLength(String newPassword) {
+        if (newPassword.length() > 7) {
+            return passwordChecks(newPassword);
+        } else {
+            System.out.println(" ");
+            System.out.println("\nSorry the password" + " " + newPassword + " " + "does not meet the requirements");
+            return false;
+        }
+    }
+
+    private static boolean passwordChecks(String newPassword) {
+        boolean numericalCheck = false;
+        boolean capitalCheck = false;
+        boolean lowerCaseCheck = false;
+        char c;
+        // For the password to be accepted all conditions must be true otherwise it will not be.
+        for (int i = 0; i < newPassword.length(); i++) {
+            c = newPassword.charAt(i);
+            if (Character.isDigit(c)) {
+                numericalCheck = true;
+            }
+            if (Character.isLowerCase(c)) {
+                lowerCaseCheck = true;
+            } else if (Character.isUpperCase(c)) {
+                capitalCheck = true;
+            }
+            if (numericalCheck && lowerCaseCheck && capitalCheck) {
+                return true;
+            }
+        }
+        return false;
+    }
+
+
+    public static String getNewUserBalance() {
+        String userOpenBalance;
+        boolean isItAFloat;
+        String response;
+        float newBalance;
+        do {
+            isItAFloat = true;
+            System.out.println("\nPlease enter your opening balance (xx.xx): ");
+            userOpenBalance = sc.nextLine();
+            try {
+                newBalance = Float.parseFloat(userOpenBalance);
+                System.out.println("New Account Balance:" + "£" + newBalance);
+            } catch (Exception e) {
+                isItAFloat = false;
+                System.out.println("\nThis is not a correct value input");
+            }
+        } while (!isItAFloat);
+        do {
+            System.out.print("\nIs this correct (Y/N)?: ");
+            response = sc.nextLine().toUpperCase();
+            if (response.equals("N")) {
+                System.out.print("Ok,what would you like the update opening balance to be?: ");
+                userOpenBalance = sc.nextLine();
+                try {
+                    newBalance = Float.parseFloat(userOpenBalance);
+                    System.out.println("New Account Balance:" + "£" + newBalance);
+                } catch (Exception e) {
+                    isItAFloat = false;
+                    System.out.println("\nThis is not a correct value input");
+                }
+                System.out.print("\nIs this correct (Y/N)?: ");
+                response = sc.nextLine().toUpperCase();
+            }
+        } while (!response.equals("Y"));
+        System.out.println("New Account Balance:" + "£" + userOpenBalance);
```

Comment:
This is also a bit redundant, you can just put the currency into the first string